torch/nn/modules/linear.py

```diff
@@ -181,7 +181,8 @@ def extra_repr(self) -> str:
 
 
 class LazyLinear(LazyModuleMixin, Linear):
-    r"""A :class:`torch.nn.Linear` module with lazy initialization.
+    r"""A :class:`torch.nn.Linear` module that infers the `in_features` parameter by
+    running a simple input through the network.
```

Comment:
Not sure about this change. The same thing is mentioned in the paragraph just below. Also not sure what the "simple" here refers to?

Reply:
gave it a twist :)